debug: cpu_load: Add support for nRF54L and deprecate cpu_load_init (#24055)
nrfconnect/sdk-nrf: doc/nrf/libraries/debug/cpu_load.rst

Configuration
*************

> To use the module, enable :kconfig:option:`CONFIG_NRF_CPU_LOAD` option.

Reviewer: ```suggestion
To use the module, enable the :kconfig:option:`CONFIG_NRF_CPU_LOAD` Kconfig option.
```

The module allows you to configure the following options in Kconfig:

> * :kconfig:option:`CONFIG_NRF_CPU_LOAD_CMDS` option to enable shell commands
> * :kconfig:option:`CONFIG_NRF_CPU_LOAD_LOG_PERIODIC` option to enable periodic logging of the
>   CPU load. :kconfig:option:`CONFIG_NRF_CPU_LOAD_LOG_INTERVAL` configures the interval.
> * :kconfig:option:`CONFIG_NRF_CPU_LOAD_ALIGNED_CLOCKS` to enable the alignment of the clock sources
>   for more accurate measurement.
> * Choosing the TIMER instance for the load measurement (for example :kconfig:option:`CONFIG_NRF_CPU_LOAD_TIMER_0`)

Reviewer: ```suggestion
* :kconfig:option:`CONFIG_NRF_CPU_LOAD_CMDS` - to enable shell commands
* :kconfig:option:`CONFIG_NRF_CPU_LOAD_LOG_PERIODIC` - to enable periodic logging of the
  CPU load.
   You can use the :kconfig:option:`CONFIG_NRF_CPU_LOAD_LOG_INTERVAL` Kconfig option to configure the interval of the logging.
* :kconfig:option:`CONFIG_NRF_CPU_LOAD_ALIGNED_CLOCKS` - to enable the alignment of the clock sources
  for more accurate measurement.
* ``CONFIG_NRF_CPU_LOAD_TIMER_*`` - to choose the TIMER instance for the load measurement (for example,  :kconfig:option:`CONFIG_NRF_CPU_LOAD_TIMER_0`)
```

Configuration
*************

To use the module, enable the :kconfig:option:`CONFIG_NRF_CPU_LOAD` Kconfig option.

The module allows you to configure the following options in Kconfig:

> * :kconfig:option:`CONFIG_NRF_CPU_LOAD_CMDS` - to enable shell commands
> * :kconfig:option:`CONFIG_NRF_CPU_LOAD_LOG_PERIODIC` - to enable periodic logging of the
>   CPU load.
>    You can use the :kconfig:option:`CONFIG_NRF_CPU_LOAD_LOG_INTERVAL` Kconfig option to configure the interval of the logging.
> * :kconfig:option:`CONFIG_NRF_CPU_LOAD_ALIGNED_CLOCKS` - to enable the alignment of the clock sources
>   for more accurate measurement.
> * ``CONFIG_NRF_CPU_LOAD_TIMER_*`` - to choose the TIMER instance for the load measurement (for example,  :kconfig:option:`CONFIG_NRF_CPU_LOAD_TIMER_0`)

Reviewer 2: ```suggestion
* :kconfig:option:`CONFIG_NRF_CPU_LOAD_CMDS` - To enable shell commands
* :kconfig:option:`CONFIG_NRF_CPU_LOAD_LOG_PERIODIC - To enable periodic logging of the
  CPU load.
   You can use the :kconfig:option:`CONFIG_NRF_CPU_LOAD_LOG_INTERVAL` Kconfig option to configure the interval of the logging.
* :kconfig:option:`CONFIG_NRF_CPU_LOAD_ALIGNED_CLOCKS` - To enable the alignment of the clock sources
  for more accurate measurement.
* ``CONFIG_NRF_CPU_LOAD_TIMER_*`` - To choose the TIMER instance for the load measurement (for example,  :kconfig:option:`CONFIG_NRF_CPU_LOAD_TIMER_0`)
```

> 

Reviewer: The Configuration section is missing information about how to enable the library. It is required by the [template guidelines](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/templates/library_template_README.html). Please add it.

The current contents of Configuration are missing links to Kconfig options -- please add them using `:kconfig:option:` formatting.

Author: added
